### PR TITLE
[WIP] (GH-1186) (1password) Changed the installer paramters a bit

### PR DESCRIPTION
**Pull request by mkevenaar**
## Description
1Password7 has changed the parameters for install, they changed whole installer. ref #1186

## Motivation and Context
New silent params.

## How Has this Been Tested?
This one is still WIP, I believe I got an return code 5.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:
- [X] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [ ] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

**Comment by AdmiringWorm** [comment]
I was surprised when I noticed it was successful, but found it tested the wrong version (tested 1password4).

I'll have a look and see if I can find out what's happening with the permission denied error you mentioned.

**Comment by mkevenaar** [comment]
> I was surprised when I noticed it was successful, but found it tested the wrong version (tested 1password4).
So was I. 

Thanks for doing this!

File: automatic/1password/update.ps1 [hunk 1]
```diff
@@ -10,16 +10,19 @@ function global:au_SearchReplace {
...
 
-function global:au_AfterUpdate {
+function global:au_BeforeUpdate {
   . "$PSScriptRoot/update_helper.ps1"
   if ($Latest.PackageName -eq '1password4') {
     removeDependencies ".\*.nuspec"
+    $Latest.SilentArgs = '/VERYSILENT /NORESTART /SUPPRESSMSGBOXES /SP- /LOG=`"$($env:TEMP)\$($env:chocolateyPackageName).$($env:chocolateyPackageVersion).InnoInstall.log`"'
   } else {
     addDependency ".\*.nuspec" 'dotnet4.6.2' '4.6.01590.20170129'
+    $Latest.SilentArgs = '-s -l=`"$($env:TEMP)`"'
```
Comment (fsouza) [inline automatic/1password/update.ps1]: It sounds like ``-l`` should point to a file, not a folder?

```suggestion
    $Latest.SilentArgs = '-s -l=`"$($env:TEMP)\$($env:chocolateyPackageName).$($env:chocolateyPackageVersion).InnoInstall.log`"'
```

From the installer ``--help`` output:

```
OPTIONS:
    -i, --install_path <INSTALL_PATH>    Optional: Custom install location
    -l, --log_path <LOG_PATH>            Optional: Custom log file location
```

**Comment by majkinetor** [comment]
None of the solutions worked, I made something, local tests are good. Closing this.

**Pull request closed without merging**